das_client/settings/lib/src/repository/settings_config_repository_impl.dart

```diff
@@ -0,0 +1,51 @@
+import 'package:logging/logging.dart';
+import 'package:settings/src/api/dto/logging_setting_dto.dart';
+import 'package:settings/src/api/settings_api_service.dart';
+import 'package:settings/src/data/local/ru_feature_database_service.dart';
+import 'package:settings/src/repository/settings_config_repository.dart';
+
+final _log = Logger('DasConfigRepositoryImpl');
+
+class SettingsConfigRepositoryImpl implements SettingsConfigRepository {
+  SettingsConfigRepositoryImpl({required this.apiService, required this.databaseService}) {
+    init();
+  }
+
+  final SettingsApiService apiService;
+  final RuFeatureDatabaseService databaseService;
+
+  LoggingSettingDto? _loggingSetting;
+
+  void init() async {
+    final success = await loadSettings();
+    if (!success) {
+      Future.delayed(Duration(minutes: 1)).then((value) {
```

Comment:
Nitpick: Retry Dauer `Duration(minutes: 1)` in statische Variable oder konfigurierbar?